rsearch/client.go

```diff
@@ -0,0 +1,47 @@
+package rsearch
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+)
+
+// SearchResource connects to instance of a server
+// and resolves SearchRequest
+func SearchResource(config Config, req SearchRequest) SearchResponse {
+	// TODO need to make url configurable
+	url := config.Server.Host + ":" + config.Server.Port
+	data := []byte(`{ "tag" : "` + req.Tag + `"}`)
```

Comment:
Something's happening with data, but I have no idea what's going on. Some comments would be really useful.